Makefile.in

```diff
@@ -0,0 +1,134 @@
+#
+# Makefile rules for the datastructures package
+#
+# This requires GNU make (just as the main GAP does).
+#
+# For simple packages one should only need to modify the PKG_SO_NAME
+# and the SOURCES variable.
+#
+PKG_SO_NAME = datastructures.so
+
+SOURCES =
+SOURCES += src/avl.c
+SOURCES += src/binaryheap.c
+SOURCES += src/datastructures.c
+SOURCES += src/hashfunctions.c
+SOURCES += src/hashmap.c
+SOURCES += src/pairingheap.c
+SOURCES += src/skiplist.c
+SOURCES += src/uf.c
+
+########################################################################
+# Do not edit below (unless you know what you're doing.
+########################################################################
+all: bin/@GAPARCH@/$(PKG_SO_NAME)
+.PHONY: all
+
+default: all
+.PHONY: default
+
+########################################################################
+# Some variables (that should come from GAP's configure'd setup
+########################################################################
+GAPPATH = @GAPPATH@
+GAPARCH = @GAPARCH@
+
+GAP=$(GAPPATH)/bin/gap.sh
+
+MKDIR_P = /bin/mkdir -p
+GAC = @GAPPATH@/bin/@GAPARCH@/gac
+
+########################################################################
+# Object files
+########################################################################
+
+# OBJS shall contain the names of all object files that constitute GAP.
+# So turn all src/*.c and gen/*.c file names in SOURCES into obj/*.lo
+OBJS = $(patsubst src/%.c,obj/%.lo,$(SOURCES))
```

Comment:
The comment does not match the code

Reply:
The real question here is whether we want to put in running ward, in which case I'd change to code to match the comment. Do you have any opinion on whether we put ward into this system?